scss/base/_variables.scss

```diff
@@ -131,6 +131,8 @@ $zindex-ad-top-bar:                       12600 !default;
 $zindex-popover:                          12600 !default;
 $SUPER_Z_INDEX:                           20000 !default; //in core, here for bird's eye view tracking
 $zindex-fullscreen:                  2147483647 !default; //in screenfull, here for bird's eye view tracking
+$zindex_1000:                              1000 !default;
+$zindex_1001:                              1001 !default;
```

Comment:
While this is technically a constant, putting the value in the name doesn't seem useful. Also, all the other constants use '-' not underscore. Can you rename these to 
zindex-state-screen and zindex-vr-controls

Reply:
/+

(ok, sure. But you may have a problem with logic of the names when you want to use the same values for z-index in other files.)